src/components/MagicCodeInput.js

```diff
@@ -293,7 +293,14 @@ function MagicCodeInput(props) {
                         key={index}
                         style={[styles.w15]}
                     >
-                        <View style={[styles.textInputContainer, focusedIndex === index ? styles.borderColorFocus : {}, props.hasError || props.errorText ? styles.borderColorDanger : {}]}>
+                        <View
+                            style={[
+                                styles.textInputContainer,
+                                {height: styles.magicCodeInputContainer.minHeight - styles.textInputContainer.borderBottomWidth},
```

Comment:
@rushatgabhane  Because this style only applies to a single-use component. So I think we should define it in component as the guideline

```

## When to Create a New Style

If we need some minimal set of styling rules applied to a single-use component then it's almost always better to use an array of helper styles rather than create an entirely new style if it will only be used once. Resist the urge to create a new style for every new element added to a screen. There is a very good chance the style we are adding is a "single-use" style.


// Bad - Since we only use this style once in this component
const TextWithPadding = props => (
    <Text style={styles.textWithPadding}>
        {props.children}
    </Text>
);

// Good
const TextWithPadding = props => (
    <Text
        style={[
            styles.p5,
            styles.noWrap,
        ]}
    >
        {props.children}
    </Text>
);


```